### DISCUSSION TITLE
How to delete or hide a contribution from my profile?

### DISCUSSION THREAD

**Discussion by kaleshart** [community/community#81576] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

just suggest somthing

**Comment by kapilnish** [comment, marked as answer]
Here are general steps you can follow for common platforms:

GitHub:
Go to the repository where the contribution was made.
Navigate to the specific file or commit you want to delete.
Click on the file or commit, and you should see an "Edit" or "Delete" option.
Make the necessary changes or delete the file/commit.
Commit the changes, and they will be reflected in the repository.
GitLab:
Go to the repository where the contribution was made.
Navigate to the specific file or commit you want to delete.
Click on the file or commit, and you should see an "Edit" or "Delete" option.
Make the necessary changes or delete the file/commit.
Commit the changes, and they will be reflected in the repository.

**Reply by devmuhib009** [reply]
Thanks for the suggestion. This will help other people as well.

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
